bciers/libs/components/src/form/widgets/CheckboxWidget.tsx

```diff
@@ -12,6 +12,7 @@ const CheckboxWidget: React.FC<WidgetProps> = ({
   uiSchema,
 }) => {
   const alignment = uiSchema?.["ui:options"]?.alignment || "center";
+  const customLabel = uiSchema?.["ui:options"]?.label || label;
```

Comment:
This mixes 2 label props (RJSF doesn't make it very clear)
- RJSF defines `label` prop on the UiSchema as having the type `boolean | undefined` (it's meant to be a toggle to show the title label or not).
- the `label` prop in the WidgetProps is the `title` property in the JsonSchema ("Assertion Statement" in this case)

We can add our custom properties to the UiSchema instead, and retrieve it with the RJSF helper from the ui schema: 
```suggestion
  const {checkboxLabel = ""} = getUiOptions(uiSchema, registry.globalUiOptions)
```
There's an example of that in the ArrayFieldTemplate